gulp/styles.js

```diff
@@ -3,6 +3,8 @@
 var path = require('path');
 var gulp = require('gulp');
 var conf = require('./conf');
+var gulpif = require('gulp-if');
```

Comment:
I don't think it's implemented in this project. :S 
I tried and got an error saying that $.if is not a function